### PR TITLE
Russian version of template

### PR DESCRIPTION
**Pull request by nerilin**
Hi!
I really like the Arc template and I use it in my work.
And I would like to slightly popularize it in the Russian-speaking community, so I prepared its translation with minimal adaptation.
I would appreciate any comments.

### CODE REVIEW

**Comment by gernotstarke** [comment]
Great news, thank you very much for your effort.

I propose to check and merge your PR in two phases:

1. check translation - we're lucky to have some RU speaking people within reach :-)
2. verify that our build process can cope with cyrillic font - we have never pushed our toolchain
in that direction... (I hope @rdmueller can take up that part)

please give us a couple of days - christmas is approaching and project-stress levels are above normal...

**Comment by gernotstarke** [comment]
btw - are you interested in translating an arc42 example documentation too - it helped acceptance in the DE and EN area a lot... maybe you have a look at hsc.aim42.org (and click on "architecture") to find an open source example.

File: RU/asciidoc/src/10_quality_scenarios.adoc [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+Так как качественные требования оказывают большое влияния на принятие архитектурных решений, вы должны зафиксировать качества системы,
+достижение которых важно для каждого из стейкхолдеров, в конкретной и измеримой форме. 
+****
+
+=== Дерево атрибутов качества
+
+[role="arc42help"]
+****
+.Содержание
+Деревот атрибутов качества (также называемое ATAM - Architecture Tradeoff Analysis Method) с описанием сценариев проверки качества
```
[the remark this answers is not in the document]
Comment (gernotstarke) [reply RU/asciidoc/src/10_quality_scenarios.adoc]: thank you so much for your input - I'm too sorry I don't understand the comment...

regards, Gernot

[the remark this answers is not in the document]
Comment (highsource) [reply RU/asciidoc/src/10_quality_scenarios.adoc]: The word "Деревот" is misspelled, should be written as "Дерево", without the last "т".

[the remark this answers is not in the document]
Comment (nerilin) [reply RU/asciidoc/src/10_quality_scenarios.adoc]: Thank you, this is a typo, I'll fix it

**Comment by nerilin** [comment]
> btw - are you interested in translating an arc42 example documentation too - it helped acceptance in the DE and EN area a lot... maybe you have a look at hsc.aim42.org (and click on "architecture") to find an open source example.

Yeah, It sounds great. I'll try to do it after the holidays.

**Comment by highsource** [comment]
I'm available for quick cross-reads or checks, but I won't be able to do larger work packages, sorry.

**Comment by rdmueller** [comment]
@highsource quick cross checks already help a lot!

**Comment by gernotstarke** [comment]
thanx for your contribution. Although I'm not able to fully verify your translation, I'll simply merge your PR.

Next steps should be that we

1. generate the output document types (html, pdf, etc) from your input (@rdmueller - could you please work your build-magic on the RU version...)
2. update the arc42.org website to reflect this major enhancement
3. advertise the RU language version on Twitter :-)

**Comment by nerilin** [comment]
Thank you a lot! :)

**Comment by gernotstarke** [comment]
Nikita, what's your twitter handle (in case you have one...)? I'd like to tweet about your translation and give you the credits! you may also reply via email (gs@gernotstarke.de) or twitter-dm (@gernotstarke)

thx, Gernot

**Comment by nerilin** [comment]
My handle is @ErilinNik. Unfortunately, I can't send you dm in twitter right now.

Thank you!

**Pull request merged**